docs/DeveloperGuide.adoc

```diff
@@ -273,94 +275,121 @@ Refer to the guide <<Testing#, here>>.
-
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="59%",cols="4*",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
+|&#11088;&#11088;&#11088;
+|new user
+|see usage instructions
+|refer to instructions when I forget how to use the App.
+
+|&#11088;&#11088;&#11088;
+|user
+|create a new meal entry
+|keep track of what I ate today.
+
+|&#11088;&#11088;&#11088;
+|user
+|see how many calories I have left in today's budget
+|know what I can eat later, and stay in budget.
+
+|&#11088;&#11088;&#11088;
+|user
+|be alerted if I missed a meal
+|be reminded to stick to my meal plans.
+
+|&#11088;&#11088;&#11088;
+|user
+|change my daily calorie budget
+|better reflect my new lifestyle requirements.
+
+|&#11088;&#11088;
+|user
+|modify my previous meal entries
+|fix missing or inaccurate information.
+
+|&#11088;&#11088;
+|user
+|delete my previous meal entries
+|undo mistakenly added meal entries.
+
+|&#11088;&#11088;
+|user
+|generate a summary of meals over the past month
+|review my overall progress in keeping to my meal plan.
+
+|&#11088;&#11088;
+|user
+|view meal suggestions that fit in my remaining budget.
+|figure out what I can eat while keeping to my plan.
+
+|&#11088;
+|user
+|create a new meal entry based of a meal I commonly eat
+|keep track of a meal I eat often.
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
-
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
-
-[discrete]
-=== Use case: Delete person
+// [discrete]
+=== Use Case: Add meal entry
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-+
+1.	User enters meal information.
+2.	System adds entry into database.
+3.	System shows updated calorie budget for the day.
 Use case ends.
 
 *Extensions*
 
-[none]
-* 2a. The list is empty.
-+
-Use case ends.
-
-* 3a. The given index is invalid.
+* 1a.	The given meal entry has insufficient information (calories, meal name, etc.)
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 1a1.	CaloFit shows an error message.
 +
-Use case resumes at step 2.
+Use case restarts at step 1.
```

Comment:
Shouldn't the use case ends here? You are going to be stuck in a loop forever if you don't have the necessary information for any meal.